pkg/process/procutil/process_linux.go

```diff
@@ -530,6 +531,27 @@ func (p *Probe) getLinkWithAuthCheck(pidPath string, file string) string {
 	return str
 }
 
+// getFDCount gets num_fds from /proc/(pid)/fd
+func (p *Probe) getFDCount(pidPath string) int32 {
+	path := filepath.Join(pidPath, "fd")
+
+	if err := p.ensurePathReadable(path); err != nil {
+		return 0
```

Comment:
Would it be useful to return `-1` when there's an error during the collection so we can differentiate absence of value and errors?

Reply:
I updated to return -1, thanks.